src/main/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/output/PriceSheet.kt

```diff
@@ -27,38 +35,42 @@ abstract class PriceSheet(val sheet: Sheet, private val header: Header) {
         // TODO need to add version as well.
     }
 
-    private fun writeRow(row: Row, rowValue: RowValue){
+    private fun writeRow(row: Row, rowValue: RowValue, isShaded: Boolean = false, showNotes: Boolean = true){
         with(rowValue) {
-            row.addCell(0, ref)
-            row.addCell(1, pickUp)
-            row.addCell(2, pickUpLocationType)
-            row.addCell(3, dropOff)
-            row.addCell(4, dropOffLocationType)
-            row.addCell(5, pickUpDate)
-            row.addCell(6, pickUpTime)
-            row.addCell(7, dropOffDate)
-            row.addCell(8, dropOffTime)
-            row.addCell(9, vehicleReg)
-            row.addCell(10, prisonNumber)
-
-            priceInPence?.let {
-                row.createCell(11).setCellValue(it.toDouble() / 100)
-            } ?: row.createCell(11).setCellValue("NOT PRESENT")
-
-            row.addCell(12, billable)
+            row.addCell(0, ref, isShaded)
+            row.addCell(1, pickUp, isShaded)
+            row.addCell(2, pickUpLocationType, isShaded)
+            row.addCell(3, dropOff, isShaded)
+            row.addCell(4, dropOffLocationType, isShaded)
+            row.addCell(5, pickUpDate, isShaded)
+            row.addCell(6, pickUpTime, isShaded)
+            row.addCell(7, dropOffDate, isShaded)
+            row.addCell(8, dropOffTime, isShaded)
+            row.addCell(9, vehicleReg, isShaded)
+            row.addCell(10, prisonNumber, isShaded)
+            row.addCell(11, priceInPounds ?: "NOT PRESENT", isShaded)
+            row.addCell(12, billable, isShaded)
+            row.addCell(13, if(showNotes) notes else "", isShaded)
         }
     }
 
-    fun writeMoveRow(price: MovePrice) = writeRow(createRow(), RowValue.forMovePrice(price))
+    fun writeMoveRow(price: MovePrice, isShaded: Boolean, showNotes: Boolean = true) = writeRow(createRow(), RowValue.forMovePrice(price), isShaded, showNotes )
```

Comment:
I have to admit I am getting a bit lost in the various functions for adding data.  Perhaps add some documentation?